webf/lib/src/module/local_storage.dart

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2022-present The WebF authors. All rights reserved.
+ */
+
+import 'dart:async';
+import 'dart:math';
+import 'dart:convert';
+import 'package:path/path.dart' as path;
+import 'package:hive/hive.dart';
+import 'package:webf/foundation.dart';
+import 'package:webf/module.dart';
+
+class LocalStorageModule extends BaseModule {
+  @override
+  String get name => 'LocalStorage';
+
+  static String getBoxKey(ModuleManager moduleManager) {
+    String origin = moduleManager.controller.origin;
+    final bytes = utf8.encode(origin);
+    final base64Str = base64.encode(bytes);
+    return '_webf_${base64Str.substring(0, min(base64Str.length, 30))}';
```

Comment:
I think it's better to increase the limitation or remove the limitation when using the origin as key. 30 is too small.
The following two URLs would return the same key.
`https://developer.apple.com` `https://developer.apple.com.cn` => `aHR0cHM6Ly9kZXZlbG9wZXIuYXBwbG`

Reply:
The `hive` support the max length of key is `255`.
https://docs.hivedb.dev/#/more/limitations

Reply:
done